packages/@react-spectrum/text/src/Keyboard.tsx

```diff
@@ -26,7 +26,7 @@ function Keyboard(props: KeyboardProps, ref: DOMRef) {
   let domRef = useDOMRef(ref);
 
   return (
-    <kbd {...filterDOMProps(otherProps)} {...styleProps} dir="ltr" ref={domRef}>
+    <kbd {...filterDOMProps(otherProps)} {...styleProps} ref={domRef}>
```

Comment:
This is because with real logical properties, `dir` will affect which side `padding-inline-start` applies to as well as the text itself. Changed to use `unicode-bidi: plaintext` instead.